### PR TITLE
Fix typo parsing module reqs

### PR DESCRIPTION
**Pull request by r4f4**
Signed-off-by: Rafael Fonseca <r4f4rfs@gmail.com>

### CODE REVIEW

**Comment by igor-raits** [comment]
@r4f4 does libmodulemd reject stream list like `[-1, 2]`? Or is it just that people SHOULD NOT do that because it is pointless?

**Comment by r4f4** [comment]
@ignatenkobrain right now it doesn't reject but it's a bug in libmodulemd according to @sgallagher.

**Comment by igor-raits** [comment]
once it does, let's bump version of libmodulemd and do this.

File: main.c [hunk 1]
```diff
@@ -45,22 +45,17 @@ parse_module_requires (Pool       *pool,
...
+       * positive and negative (build)requires for a hash entry */
       Id req_neg = 0, req_pos = 0;
+      Id *r = (*ss && *ss[0] == '-') ? &req_neg : &req_pos;
       for (; *ss; ss++)
         {
           const char *s = *ss;
-
-          Id *r;
           if (s[0] == '-')
             {
```
Comment (igor-raits) [inline main.c]: remove these

**Comment by r4f4** [comment]
Ok, so I'll update the patch to just fix the typo for now.

**Comment by sgallagher** [comment]
> @ignatenkobrain right now it doesn't reject but it's a bug in libmodulemd according to @sgallagher.

Yes, libmodulemd should be validating this input. I'm going to be reworking that interface in API 2.0 to make it so that it's rejected properly. @r4f4 please open an issue in https://github.com/fedora-modularity/libmodulemd/issues

**Comment by r4f4** [comment]
ping

**Pull request merged**